### DISCUSSION TITLE
UV available only as a daily param?

### DISCUSSION THREAD

**Discussion by drmrbrewer** [open-meteo/open-meteo#250] [General]
I note with interest that [UV is now available](https://openmeteo.substack.com/p/precipitation-probability-uv-index) as a parameter in the [Weather API](https://open-meteo.com/en/docs).  But only as a daily rather than hourly parameter?  Is it possible that in future it might be hourly?  Thanks!

**Comment by patrick-zippenfenig** [comment]
Data is available already, but it is not documented. You can use `hourly=uv_index,uv_index_clear_sky` to get hourly UV data. However I am not sure if it is wise to actively promote this option

https://api.open-meteo.com/v1/forecast?latitude=52.52&longitude=13.41&hourly=uv_index,uv_index_clear_sky

**Reply by drmrbrewer** [reply]
Ok, thanks.  Why not wise?

**Reply by patrick-zippenfenig** [reply]
The recommended default presentation is the daily maximum. Even the icon, colours, look&feel is defined by the WHO. With hourly data, developers could start to build graphs with hourly information which could give user the impression that given a certain hour of the day it is "safe" not to use skin protection.

**Reply by drmrbrewer** [reply]
I kinda get that, though many APIs already provide hourly UV forecast data.

**Reply by patrick-zippenfenig** [reply]
Maybe I will add another drop down and "hide" it... a bit like solar radiation variables

**Reply by drmrbrewer** [reply]
That sounds like a plan.

**Reply by patrick-zippenfenig** [reply]
New category will be called "Additional Variables" see https://github.com/open-meteo/open-meteo/commit/41ac846801a3e83047ec9979b3ef3c049fc04a74
